frontend/src/components/common/text-area/TextArea.tsx

```diff
@@ -11,6 +11,23 @@ const TextArea = ({ highlight, singleRow, placeholder, value, onChange }:
   { highlight: boolean; singleRow?: boolean; placeholder: string; value: string; onChange: Function }) => {
 
   const minRows = singleRow ? 1 : 5
+  const textAreaRef = useRef<HTMLTextAreaElement | null>(null)
+
+  useEffect(() => {
+    // ignores Enter key presses as it triggers a resize on textarea
+    if (singleRow && textAreaRef.current) {
+      textAreaRef.current.onkeydown = (e) => {
+        if (e.keyCode === 13) {
+          e.preventDefault()
+        }
+      }
+    }
+  }, [textAreaRef.current, singleRow])
```

Comment:
hmm since we're already preventing line breaks in the `onChange` event, could we do without this?

Reply:
it's a workaround for the weird behaviour exhibited by the autosize textarea library. Even if line breaks are removed, pressing enter will trigger a resize and the height will increase until another key is pressed.